bears/general/AnnotationBear.py

```diff
@@ -18,14 +18,17 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
         The Bear searches for valid comments and strings and yields their
         ranges as SourceRange objects in HiddenResults.
 
-        :param language:        Language to be whose annotations are to be
-                                searched.
-        :param coalang_dir:     external directory for coalang file.
-        :return:                HiddenResults containing a dictionary with
-                                keys as 'strings' or 'comments' and values as
-                                a tuple of SourceRanges of strings and
-                                a tuple of SourceRanges of comments
-                                respectively.
+        :param language:
+            Language to be whose annotations are to be searched.
+        :param coalang_dir:
+            External directory for coalang file.
+        :return:
+            One HiddenResult containing a dictionary with keys being 'strings'
+            or 'comments' and values being a tuple of SourceRanges pointing to
+            the strings and a tuple of SourceRanges pointing to all comments
+            respectively. The ranges do include string quotes or the comment
+            starting separator but not anything before (e.g. a ``u`` when using
```

Comment:
this means that ``u`` is not included right ? in that case i fel like you need to say it explictly (this kind of suggests that ``u`` is included)